pkg/config/config.go

```diff
@@ -228,7 +228,7 @@ func NewMountSpec() *types.MountSpec {
 			}, {
 				Mountpoint: constants.EfiDir,
 				Device:     fmt.Sprintf("PARTLABEL=%s", constants.EfiPartName),
-				Options:    []string{"ro", "defaults"},
+				Options:    []string{"rw", "defaults"},
```

Comment:
Having EFI writable is aligned with SL Micro and it also simplifies a lot the `08_boot_assessment.yaml` file. I have mixed feelings about this, I am not this is better.

Reply:
Yep, was also hesitant about this, but it is fairly common. :shrug: